Java/core/src/main/java/com/amazon/randomcutforest/tree/AbstractCompactRandomCutTree.java

```diff
@@ -60,7 +60,8 @@
 
     protected INodeStore nodeStore;
     protected IPointStoreView<Point> pointStore;
-    protected AbstractBoundingBox<Point>[] cachedBoxes;
+    protected IBoxCache<Point> cacheManager;
```

Comment:
Could we just call this `boxCache` to match the type?

Reply:
Done